### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AntonBessarab**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,6 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    low_phrase = phrase.lower()
+    result_string = low_phrase.count(letter.lower())
+    return result_string
```
Comment (EdAlekseiev) [inline app/main.py:19]: rewrite this function to one line with using method `count` of `string`

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (AntonBessarab) [reply app/main.py]: > rewrite this function to one line with using method `count` of `string`

Done

Comment (AntonBessarab) [reply app/main.py]: Should I create a new pull request ?

**Review by AntonBessarab — COMMENTED** [review]
Like this ?

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**